app/controllers/api/v1/contacts_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class ContactsController < ApiController
+      def create
+        Contact.create!(contact_params)
+        ContactMailer.contact(
+          contact_params[:email],
```

Comment:
Should this be user's email?
Are we storing any info about the creator of the Contact?

Reply:
Not a must.. but keep in mind you can move the `create` and `send_email` methods to a Service object. This would help you to keep your controller methods skinny

Reply:
No, it should be the param. I should have skipped the authentication though. I just realized that

Reply:
I don't think in this case it would be convenient, but I'll keep it in mind for more complex logic